dev/prod/src/platform.ts

```diff
@@ -138,6 +139,7 @@ export async function configurePlatform() {
   const config: Config = await (await fetch(devConfig? '/config-dev.json' : '/config.json')).json()
   console.log('loading configuration', config)
   setMetadata(login.metadata.AccountsUrl, config.ACCOUNTS_URL)
+  setMetadata(login.metadata.LogoWayUrl, config.LOGO_WAY_URL)
```

Comment:
You need to update anticrm/server/front/src/index.ts as well, or variable will not work.

Reply:
Great, thank you!

Added